functorch/_src/aot_autograd.py

```diff
@@ -865,7 +869,16 @@ def forward(self, *args, **kwargs):
     return AOTModule()
 
 
-def aot_module_simplified(mod: nn.Module, *top_args, **top_kwargs) -> nn.Module:
+def aot_module_simplified(
+        mod: nn.Module,
+        inputs: List[torch.Tensor],
+        fw_compiler: Callable,
+        bw_compiler: Optional[Callable] = None,
+        partition_fn: Callable = default_partition,
+        decompositions: Optional[Dict] = None,
+        hasher_type=None,
+        static_argnums=None,
```

Comment:
This is also a good refactor, also factored out for ease of review: https://github.com/pytorch/pytorch/pull/89666